packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControls.tsx

```diff
@@ -182,15 +182,24 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
             </>
           )}
           {!hideFacetFilterControl && (
-            <a
+            <Button
+              variant={'text'}
               onClick={() => setShowFacetFilter(value => !value)}
-              className="TopLevelControls__querycount__facetFilterLink SRC-no-underline-on-hover"
+              startIcon={
+                <IconSvg
+                  icon={showFacetFilter ? 'close' : 'filter'}
+                  wrap={false}
+                />
+              }
+              sx={{
+                ml: 2,
+                fontWeight: 400,
+                fontSize: '14px',
+                textDecoration: 'none !important',
+              }}
             >
-              <Icon type={showFacetFilter ? 'close' : 'filter'}></Icon>
-              <span className="TopLevelControls__querycount__facetFilterLink__text">
-                {showFacetFilter ? 'Hide' : 'Show'} filters
-              </span>
-            </a>
+              {showFacetFilter ? 'Hide' : 'Show'} filters
+            </Button>
```

Comment:
Instead of using `a` and tons of classes to align icons and text, just use a `Button` with the `"text"` variant. Also, use the new icon.

Reply:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/419306ae-4ca9-429d-b761-256d18d43c37)